GothicLlamasAPI/src/test/java/com/kainos/ea/resources/ServiceTest.java

```diff
@@ -0,0 +1,93 @@
+package com.kainos.ea.resources;
+
+import com.kainos.ea.objects.CapabilitiesAndRoles;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import java.util.List;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ServiceTest {
+
+    @Mock
+    private Service wcService;
+    private CapabilitiesAndRolesDAO dao;
+
+    @Before
+    public void setup() {
+        dao = mock(CapabilitiesAndRolesDAO.class);
+        wcService = new Service(dao);
+    }
```

Comment:
Use the annotation so you don't have to manually mock it
```suggestion
    @Mock
    private Service wcService;
    @Mock
    private CapabilitiesAndRolesDAO dao;

    @Before
    public void setup() {
        wcService = new Service(dao);
    }
```